apps/api/src/app/inbox/usecases/session/session.command.ts

```diff
@@ -0,0 +1,16 @@
+import { IsDefined, IsOptional, IsString } from 'class-validator';
+import { BaseCommand } from '../../../shared/commands/base.command';
+
+export class SessionCommand extends BaseCommand {
+  @IsDefined()
+  @IsString()
+  readonly applicationIdentifier: string;
```

Comment:
Mental note, to remove the need for this next week and replace it with a public client key.